### PR TITLE
feat: auth loading state

### PR DESCRIPTION
**Pull request by filipeveronezi**
https://github.com/avitorio/outstatic/assets/48724782/dad28d70-deb4-438b-b4ac-324874dceb06

## This PR:
- Adds animate-draw to tailwind config
- Adds a loading animation to the login action

## Observations:
- I've thought about adding an artificial delay in order to play the animation consistently smooth. For example, there's basically no time for a loading state if you're not signed into your GitHub account, as it only redirects you to the GitHub page. Any opinions on that?

## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have a helpful link attached, see `contributing.md`

## Feature

- [X] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added

### CODE REVIEW

File: packages/outstatic/tailwind.config.js [hunk 1]
```diff
@@ -12,6 +12,16 @@ module.exports = {
...
+      },
+      animation: {
+        draw: 'draw 3000ms linear infinite'
+      },
+      keyframes: {
+        draw: {
+          '0%': { 'stroke-dashoffset': '0' },
+          '50%': { 'stroke-dashoffset': '4000' },
+          '100%': { 'stroke-dashoffset': '0' }
+        }
```
Comment (avitorio) [inline packages/outstatic/tailwind.config.js:24]:
```suggestion
      animation: {
        draw: 'draw 5s cubic-bezier(0,.57,1,.43) infinite'
      },
      keyframes: {
        draw: {
          '0%': { 'stroke-dashoffset': '0' },
          '49.999%': { 'stroke-dashoffset': '-3000' },
          '50%': { 'stroke-dashoffset': '4000' },
          '50.001%': { 'stroke-dashoffset': '4000' },
          '100%': { 'stroke-dashoffset': '0' }
        }
      }
```

File: packages/outstatic/tailwind.config.js [hunk 2]
```diff
@@ -12,6 +12,16 @@ module.exports = {
       colors: {
         brandblue: colors.blue[500],
         brandred: colors.red[500]
+      },
```
Comment (avitorio) [inline packages/outstatic/tailwind.config.js:15]:
```suggestion
```

We can remove these colors, they aren't being used anywhere

File: packages/outstatic/tailwind.config.js (hunk 1 above)
Comment (avitorio) [reply packages/outstatic/tailwind.config.js:24]: Making the line only move forward here, also adding some cubic-bezier for better physics 🤓

**Review by avitorio — COMMENTED** [review]
Great job, just some minor adjustments...

**Comment by filipeveronezi** [comment]
@avitorio It looks awesome!! Thanks for the tweaks.

**Review by avitorio — APPROVED** [review]
LGTM

**Pull request merged**